DaoPools/src/entities/global/globals.ts

```diff
@@ -1,3 +1,4 @@
 export const PRICE_FEED_ADDRESS = "0x1fb8a1e19abb51a1224b5fbC329E225c0d08Cf51";
 export const BNB_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
 export const WBNB_ADDRESS = "0xae13d989daC2f0dEbFf460aC112a837C89BAa7cd";
+export const REWARD_TYPE_DELEGATED = 2;
```

Comment:
Why not nat it the same way it is named in SC?

```suggestion
export const REWARD_TYPE_VOTE_DELEGATED = 2;
```